docs/test_types/rlp_tests.rst

```diff
@@ -62,3 +62,9 @@ Sections
 
 * ``in`` - json object (array, int, string) representation of the rlp byte stream (\*except values ``VALID`` and ``INVALID``)
 * ``out`` - string of rlp bytes stream
+
+When a json string starts with ``0x``, the rest of the string is interpreted as
+hex bytes, and when one starts with ``#``, the rest is interpreted as a decimal
```

Comment:
Is this `#` prefixing used in the wild in any of the existing tests (and can you link)? You don't seem to introduce this here in the PR itself (?).

Reply:
It's used in 3 places in the existing tests that I found:

  1. [_mediumint4_](https://github.com/ethereum/tests/blob/d505c07e011ab8eb1b8d71ca4c550543a04aa0fc/RLPTests/rlptest.json#L66-L69), `rlptest.json` line 67, `"#83729609699884896815286331701780722"`
  2. [_mediumint5_](https://github.com/ethereum/tests/blob/d505c07e011ab8eb1b8d71ca4c550543a04aa0fc/RLPTests/rlptest.json#L70-L73), `rlptest.json` line 71, `"#105315505618206987246253880190783558935785933862974822347068935681"`
  3. [_bigint_](https://github.com/ethereum/tests/blob/d505c07e011ab8eb1b8d71ca4c550543a04aa0fc/RLPTests/rlptest.json#L153-L156), `rlptest.json` line 154, `"#115792089237316195423570985008687907853269984665640564039457584007913129639936"`

Reply:
By the way, I searched through the repository with the regexp `"#[0-9]`, and those were the only 3 that I found.

Reply:
Do you want the docs to link to these places in the code, or, just add an example of how the `#` could be used?

Reply:
No, as you stated direct links are likely not to be updated, but an example would be helpful. Do I get this right that this is a work-around for bigint integer values?

Reply:
If so you should also make this more clear in the doc description (since for ``decimals`` there is also the non-quoted more general format).

Reply:
I have updated this to add the example of `5050` and `"#5050"` both representing the decimal number `5050`, as well as a mention of using `#` prefixes when the number is too big for `int` and would require a bigint representation.